pkg/console/webhooks.go

```diff
@@ -63,7 +64,10 @@ func IsValidHTTPMethod(method string) bool {
 }
 
 func (p *RenderedWebhook) Handle() error {
+	password := p.BasicAuth.Password
```

Comment:
prefer to add a commond method to avoid duplicate code:
```go
func (p *RenderedWebhook) Debug(msg string) {
	password := p.BasicAuth.Password
	p.BasicAuth.Password = PasswordMask
	logrus.Debugf(msg+" %+v", *p)
	p.BasicAuth.Password = password
}
```

and use it like:
```go
p.Debug("rendered webhook")
```